modules/grafana/manifests/repo.pp

```diff
@@ -0,0 +1,17 @@
+# == Class: grafana::repo
+#
+# === Parameters:
+#
+# [*apt_mirror_hostname*]
+#   Hostname to use for the APT mirror.
+#
+class grafana::repo (
+  $apt_mirror_hostname,
+) {
+  apt::source { 'grafana':
+    location     => "http://${apt_mirror_hostname}/grafana",
+    release      => 'main',
+    architecture => $::architecture,
+    key          => '418A7F2FB0E1E6E7EABF6FE8C2E73424D59097AB',
```

Comment:
I think this key should be `3803E444EB0235822AA36A66EC5FE1A937E3ACBB` because that's the key for our apt mirror rather than for the original Grafana repo.